src/v2/Apps/Auction/AuctionApp.tsx

```diff
@@ -38,6 +39,9 @@ export const AuctionApp: React.FC<AuctionAppProps> = ({
     showBuyNowTab: sale.showBuyNowTab,
   }
 
+  const { cascadingEndTimeInterval } = sale
+  const isFullBleedHeaderFixed = !cascadingEndTimeInterval
```

Comment:
Brought this up as a possible design question in Slack, but when we display this banner we can't have the full bleed header image have a fixed position.